packages/editor/src/components/parts/common/eventcode/EventCodeSelect.tsx

```diff
@@ -10,9 +11,10 @@ type EventCodeSelectProps = {
   eventCodes: EventCodeItem[];
   eventIcon: IvyIcons;
   comboboxInputProps?: FieldsetInputProps;
+  withBrowser?: boolean;
```

Comment:
instead of this flag, I could also imagine to just pipe the types as parameter and if it is undefined or an empty array just don't render the browser.
but this solution is also fine.